### ISSUE TITLE
NVDA doesn't announce cells in a selected cell range as 'selected'

### ISSUE THREAD

**Issue by AG-Zoheil** [nvaccess/nvda#18536]
**Current**
JAWS announces cells as selected, NVDA doesn't

**Steps to reproduce:**

Open https://plnkr.co/edit/50GGqPWTR8TdyMYF?open=main.js
Open NVDA/JAWS
Click a grid cell
JAWS:
Actual & Expected: cell is announced as "selected"
NVDA:
Actual: cell is not announced as "selected"
Expected: cell is announced as "selected"

Inspect the DOM of the selected cell and note the aria-selected=true is set.

Desired
Allow NVDA to announce the aria-selected attribute set on the grid cell

**Comment by seanbudd** [comment]
This is expected behaviour
